### PR TITLE
change strategic goals structure

### PR DESCRIPTION
**Pull request by anajsana**
As discussed during the last SC meeting, I've updated the SC goals structure dividing them between SC and LF Staff goals.

### CODE REVIEW

File: goals.md [hunk 1]
```diff
@@ -9,31 +9,56 @@ Based on this, we would like to bring the goals to the TODO Group for review and
...
+- Host 8 OSPOlogy sessions
+- Encourage more active participation in OSPOlogy
+
+### ➡️ Grow the community
+
+- Cultivate relationships with other communities
+
+### ➡️ Create value for all members
+
+- Run at least 2 surveys to get feedback, >80% satisfaction
```
Comment (geekygirldawn) [inline goals.md:30]: I would rephrase this to "Run at least 2 TODO Group member feedback surveys ..." This will help people understand that you aren't talking about the other survey.

File: goals.md [hunk 2]
```diff
@@ -9,31 +9,56 @@ Based on this, we would like to bring the goals to the TODO Group for review and
...
 
 - Grow TODO membership by 20%
-- Cultivate relationships with other OSPO adjacent communities like CHAOSS
-- Grow OSPO Ambassadors by 
 
-## 3 Create value for all members
+### ➡️ Create value for all members
 
+- Improve and refresh TODO Website
 - Run at least 2 surveys to get feedback, >80% satisfaction
```
Comment (geekygirldawn) [inline goals.md:53]: Same comment as above to distinguish this from the OSPO survey.

**Review by geekygirldawn — CHANGES_REQUESTED** [review]

File: goals.md (hunk 2 above)
Comment (anajsana) [reply goals.md]: Done!
